src/__tests__/index.test.ts

```diff
@@ -438,6 +438,16 @@ const vectors: Array<TestVector> = [
     coinType: 239,
     passingVectors: [{ text: 'AXaXZjZGA3qhQRTCsyG5uFKr9HeShgVhTF', hex: '17ad5cac596a1ef6c18ac1746dfd304f93964354b5' }],
   },
+  {
+    name: 'NEAR',
+    coinType: 242,
```

Comment:
slip44 id is incorrect

Reply:
Thanks. Resolved. Please review and merge.